### PR TITLE
Adds more graffiti (faction themed)

### PR DESCRIPTION
**Pull request by Twomoon-Github**
## About The Pull Request

Adds 8 more graffitis (faction themed)

## Why It's Good For The Game

Adds more variety, and factions graffitis were asked a lot

## Testing
Tested: they appear on the list, and will be made like other graffitis when selected.
graffiti+

## Changelog
:cl: Twomoon, SirRichardFrancis (KIROV)
add: Added new graffitis
imageadd: added 8 more graffitis in wall_graffiti.dmi
/:cl:

### CODE REVIEW

**Comment by Twomoon-Github** [comment]
i need to work more to polish the graffiti in general, so i deleted the old PR that didnt made the situation better and remade this one, that only adds the new graffitis, most of them made by kirov

**Comment by SirRichardFrancis** [comment]
My brother in Space Christ, you apparently forgot to include my favorite ones: commie, sad IH and menacing graffiti.

image

Truth be told, my actual favorite is ANCAP ball, but still… please add these.

**Comment by SirRichardFrancis** [comment]
Also, for changelog it's SirRichardFrancis, not KIROV. Thanks.

File: code/game/objects/items/spraypaint.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 /obj/item/spraypaint/afterattack(atom/target, mob/user as mob, proximity)
 	if(!proximity) return
 	if(istype(target,/turf))
-		var/graffititype = input("Choose what you'd like to paint") in list("Kot","Onestar","Doodle","Piss","Clown","Skull","Heart")
+		var/graffititype = input("Choose what you'd like to paint") in list("Kot","Onestar","Doodle","Piss","Clown","Skull","Heart","Excelsior","Ironhammer","Moebius",,"Neotheology","Technomancer","Aster","Ancap yes","Ancap no")
```
Comment (hyperioo) [inline code/game/objects/items/spraypaint.dm:15]:
```suggestion
		var/graffititype = input("Choose what you'd like to paint") in list("Kot","Onestar","Doodle","Piss","Clown","Skull","Heart","Excelsior","Ironhammer","Moebius","Neotheology","Technomancer","Aster","Ancap yes","Ancap no")
```

**Review by hyperioo — APPROVED** [review]

**Pull request merged**